packages/cli/src/zostso/send/as-app/SendASApp.handler.ts

```diff
@@ -37,8 +37,14 @@ export default class Handler extends ZosTsoBaseHandler {
 
         commandParameters.response.progress.endSpinner();
 
-        commandParameters.response.console.log(
-            JSON.stringify(response, null, 2)
-        );
+        commandParameters.response.console.log("\n");
+        response.tsoData.forEach((data) => {
+            if(typeof data === 'string') {
+                commandParameters.response.console.log(data);
```

Comment:
Not requesting changes here, but we might want to have a test for the case when `typeof data === 'string'` so it will still print the data as expected. I know its not very useful now, but if code is added into this `if` block in the future, we will want to make sure that the data is still printed.